### PR TITLE
`cf revisions` no longer reports EXPERIMENTAL notice [main]

### PR DESCRIPTION
**Pull request by chinigo**
## Description of the Change

This commit changes the `cf revision` and `cf revisions` commands to no longer print a notice that the command is experimental.

## Why Is This PR Valuable?

The `cf revision` and `cf revisions` commands were introduced over 4 years ago and are no longer experimental.

## Applicable Issues

n/a

## How Urgent Is The Change?

Not urgent.

## Other Relevant Parties

None

## Related PRs

- #3060 - v7 
- #3061 - v8
- #3062 - main (this PR)

### CODE REVIEW

**Review by a-b — APPROVED** [review]
LGTM

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Given that the test was there to ensure that the message was present I do not think these tests make sense anymore.
If you come back to the code and see this test, it is strange 😄

**Comment by chinigo** [comment]
- Negative test assertions removed (including one whole file that contained nothing but).
- Moved the `revisions` and `revision` command into the `APPS` section of the `cf help` output.

File: command/common/internal/help_all_display.go [hunk 1]
```diff
@@ -172,7 +173,7 @@ var ExperimentalHelpCategoryList = []HelpCategory{
 	{
 		CategoryName: "EXPERIMENTAL COMMANDS:",
 		CommandList: [][]string{
-			{"revisions", "revision", "rollback"},
```
Comment (Samze) [inline command/common/internal/help_all_display.go:175]: **issue** Revision should not be made unexperimental since it does not do anything

Comment (chinigo) [reply command/common/internal/help_all_display.go:175]: Ah you're right, it was too aggressive to move _both_ of these commands out of the EXPERIMENTAL block. I also see now that [the ticket](https://jira.eng.vmware.com/browse/TPCF-10927) mentions in passing that the `rollback` command should also not be marked EXPERIMENTAL.

I'll submit another revision to all three of these PRs that:
- Restores `revision` to `EXPERIMENTAL` status.
- Moves `rollback` _out_ of `EXPERIMENTAL` status.

**Review by a-b — APPROVED** [review]

File: command/common/internal/help_all_display.go (hunk 1 above)
Comment (chinigo) [reply command/common/internal/help_all_display.go:175]: Updated PR as promised:

- `cf revision` remains in `EXPERIMENTAL` status.
- `cf revisions` is no longer in `EXPERIMENTAL` status.
- `cf rollback` is no longer in `EXPERIMENTAL` status.

**Review by Samze — APPROVED** [review]

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**